tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -930,3 +924,91 @@ def get_volume_alert_counts():
                                          'alert_count': 0
                                          }
     return alert_counts
+
+
+def get_device_tree():
+    devicetree = {}
+    try:
+        columns = 'NAME,KNAME,PKNAME,TYPE,MOUNTPOINT,SIZE'
+        keys = columns.split(',')
+        lsblk = ("lsblk --all --bytes --noheadings "
+                 "--output='%s' --path --raw" % columns)
+        cmd = cmd_utils.Command(lsblk)
+        out, err, rc = cmd.run()
+        if not err:
+            out = out.encode('utf8')
+            devlist = map(
+                lambda line: dict(zip(keys, line.split(' '))),
+                out.splitlines()
+            )
+            block_devices = {}
+            for dev_info in devlist:
+                if dev_info["TYPE"] == "disk":
+                    # create new block device list
```

Comment:
Why create a new object here? Cant we add to the same object created at L#944 ?

Reply:
I am not creating new object here, I am iterating devlist and populating everything inside block_devices only.

Reply:
ok